src/app/catalog/actions.test.ts

```diff
@@ -0,0 +1,73 @@
+import { mockedGetRegistryV01Servers } from "@mocks/fixtures/registry_v0_1_servers/get";
+import { HttpResponse } from "msw";
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import { getServers } from "./actions";
+
+// Mock the auth to bypass authentication
+vi.mock("@/lib/api-client", async (importOriginal) => {
```

Comment:
Maybe we can avoid the mock here and use the env var mock or mock only the better auth session?